src/stactools/cli/commands/migrate.py

```diff
@@ -1,15 +1,25 @@
 import click
-
-
-def migrate(option: click.Option) -> None:
-    # TODO
-    print(option)
+from stactools.core import migrate_object
 
 
 def create_migrate_command(cli: click.Group) -> click.Command:
-    @cli.command("migrate", short_help="Migrate a STAC catalog (TODO)")
-    @click.option("--option", "-o", default=1, help="A test option")
-    def migrate_command(option: click.Option) -> None:
-        migrate(option)
+    @cli.command("migrate", short_help="Migrate a STAC object")
+    @click.argument("href")
+    @click.option(
+        "-s",
+        "--save",
+        is_flag=True,
+        help="Save migrated STAC object in original location.",
+    )
+    @click.option(
+        "--show-diff/--hide-diff",
+        default=True,
+        help=(
+            "Whether to dump diff between original and migrated object to stdout. "
+            "Defaults to --show-diff. "
+        ),
+    )
+    def migrate_command(href: str, save: bool, show_diff: bool) -> None:
```

Comment:
Probably want to add a `recursive` option to migrate a whole catalog.

Reply:
yikes. Ok yeah sorry I thought that came for free. More tests needed...